builtin/provisioners/habitat/linux_provisioner.go

```diff
@@ -0,0 +1,376 @@
+package habitat
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform/communicator"
+	"github.com/hashicorp/terraform/terraform"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+const installURL = "https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh"
+const systemdUnit = `[Unit]
+Description=Habitat Supervisor
+
+[Service]
+ExecStart=/bin/hab sup run{{ .SupOptions }}
```

Comment:
Should there be a space in here? referencing https://github.com/hashicorp/terraform/pull/22705/files#diff-fa4a691c8df4da45964dde083a5b5cf5L28

Reply:
The `.SupOptions` are set in lines 110-165 of `linux_provisioner.go`, with each one having a space prefix.  I think it's safe that way, unless we want to add a starting space in the text-template (which I'm fine with), just-in-case?

Reply:
Nah, I see now that the original space was superfluous. Just wanted to flag it in case something was missed :)